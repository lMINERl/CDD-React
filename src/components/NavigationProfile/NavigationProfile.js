import logo from '../../assets/logo.svg';
import { ProfileDefault } from '../../stories/ProfileMenu.stories';
const NavigationProfile = props => {
  return ProfileDefault(
    {
      name: 'Mohamed',
      img: logo,
      menu: [
        {
          name: 'Profile',
          navigation: '/',
          fontawesomeIcon: 'fa fa-suitcase'
        },
        {
          name: 'Settings',
          navigation: '/',
          fontawesomeIcon: 'fa fa-cog'
        },
        {
          name: 'Logout',
          navigation: '/',
          fontawesomeIcon: 'fa fa-exclamation-circle'
        }
      ]
    },
    'dark'
  );
};

export default NavigationProfile;
