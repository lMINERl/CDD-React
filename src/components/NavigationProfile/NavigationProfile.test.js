import { React, shallow } from '../../test/Configurations';

import NavigationProfile from './NavigationProfile';

describe('<NavigationProfile />', () => {
  test('renders', () => {
    const wrapper = shallow(<NavigationProfile />);
    expect(wrapper).toMatchSnapshot();
  });
});
