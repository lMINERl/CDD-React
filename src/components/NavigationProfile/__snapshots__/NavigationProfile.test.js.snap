// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationProfile /> renders 1`] = `
<div
  className="profile profile--dark mr-5"
>
  <div
    className="profile profile__img__container profile__img--dark"
  >
    <img
      alt="logo"
      className="profile__img"
      src="logo.svg"
    />
  </div>
  <div
    className="profile__text"
  >
    Mohamed
  </div>
  <i
    className="ml-auto mr-3 fa fa-angle-down"
  />
  <div
    className="profile__dropdown profile__dropdown--dark"
  >
    <div
      className="profile__dropdown__row profile__dropdown__row--dark"
    >
      <i
        className="float-left ml-2 mr-2 fa fa-suitcase"
      />
      <NavLink
        to="/"
      >
        Profile
      </NavLink>
    </div>
    <div
      className="profile__dropdown__row profile__dropdown__row--dark"
    >
      <i
        className="float-left ml-2 mr-2 fa fa-cog"
      />
      <NavLink
        to="/"
      >
         Settings
      </NavLink>
    </div>
    <div
      className="profile__dropdown__row profile__dropdown__row--dark"
    >
      <i
        className="float-left ml-2 mr-2 fa fa-exclamation-circle"
      />
      <NavLink
        to="/"
      >
        Logout
      </NavLink>
    </div>
  </div>
</div>
`;
